src/store/modules/farmer/selectors.ts

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from 'reselect';
+import { Farmer } from './types';
+import { State } from '../../types';
+
+const getFarmerEntries = (state: State) => state.farmer.farmers;
+// const getCurrentFarmerUUID = (state: State) => state.currentFarmer.currentFarmerUUID; //need to merge with dairy/master for this
+
+/************Selectors for a specific farmer ***************/
+
+/**Selector for all farmers in the store */
+export const getAllFarmers = createSelector(
+  [getFarmerEntries],
+  (farmerEntries: Farmer[]) => farmerEntries);
+
+/**Selector for a subset of fields for all farmers (used on search page) */
+export const getFormattedFarmers = createSelector(
```

Comment:
This was a good idea. It might be good to define a return type for it that you create, that way 
A) We can use that as a type across the application. I think currently you are using `FarmerType` defined somewhere else in the application, changing it to `FormattedFarmer` and moving the definition of the interface to `src/store/modules/farmer/types.ts` would be most consistent with everything else